src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -72,15 +90,24 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      *                  if `someDate` is today;
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate currentDate = LocalDate.now();
+        if (someDate.isAfter(currentDate)) {
+            return someDate + " is after " + currentDate;
+        } else if (someDate.isBefore(currentDate)) {
+            return someDate + " is before " + currentDate;
+        }
+        return someDate + " is today";
     }
 
     /**
      * Given a String representation of a date and some timezone,
      * return LocalDateTime in this timezone.
      */
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        ZoneId zoneId = ZoneId.of(zone);
+        LocalDateTime localDateTime = LocalDateTime.parse(dateInString.substring(0,
+                dateInString.length() - 1));
```

Comment:
Interesting, but I don't get why you need substring here

Reply:
Because we have in tests string with `Z` int the end
![image](https://user-images.githubusercontent.com/52455671/103337466-2ad9fe00-4a84-11eb-8f09-9711b8b75c1c.png)

Reply:
you can declare a ZonedDateTime variable and parse there without substring. And then use .toLocalDateTime to return what you need. You can stay with this solution, but my suggestion is a bit more readable.